spec/integration/logging_spec.rb

```diff
@@ -21,7 +21,7 @@
   let(:lines) { @log_output.readlines }
 
   let(:config_flags) {
-    %W[--nodes #{uri} --no-host-key-check --format json --modulepath #{modulepath} --password #{password}]
+    %W[--targets #{uri} --no-host-key-check --format json --modulepath #{modulepath} --password #{password}]
```

Comment:
I wonder if updating all spec tests to use this flag now would be part of this ticket? I maybe it should be part of https://github.com/puppetlabs/bolt/issues/1376 or its own separate thing. Either way I think we will need to do it eventually.